modules/ipc/ipcProviderBackend.js

```diff
@@ -337,14 +337,14 @@ class IpcProviderBackend {
         })
         .catch((err) => {
 
-            err = this._makeErrorResponsePayload(payload || {}, {
+            var returnErr = this._makeErrorResponsePayload(payload || {}, {
```

Comment:
But thats actually part of another PR: https://github.com/ethereum/mist/pull/1421 i will change it there